packages/lesswrong/server.js

```diff
@@ -65,13 +67,16 @@ import './lib/collections/users/validate_login.js';
 import './lib/collections/users/callbacks.js';
 import './lib/collections/bans/callbacks.js';
 import './lib/collections/posts/tableOfContents.js';
-import './lib/collections/localgroups/callbacks.js';
+if (getSetting('events', true)) {
+  import './lib/collections/localgroups/callbacks.js';
+}
 
 import './lib/collections/revisions/resolvers.js';
 import './lib/collections/posts/serverSchema.js';
 
-import './lib/events/server.js';
-
+if (getSetting('events', true)) {
+  import './lib/events/server.js';
```

Comment:
This file doesn't do what you think it does; it's the bridge that puts events (like clicks and logins) into Intercom.